ci/playbooks/tcib/tcib.yml

```
@@ -44,7 +44,7 @@
         - "'change_url' in item"
 
     - name: Build containers when needed
-      when: "'os-net-config' not in zuul_change_list"
+      when: "zuul_change_list is defined and 'os-net-config' not in zuul_change_list"
```

```suggestion
      when:
        -  zuul_change_list is defined
        - "'os-net-config' not in zuul_change_list"
```
slightly more readable - and doing the same

---

thanks, updated the PR